### PR TITLE
Feature 317 timeline resource and property editor clean up

### LINKED ISSUE #326: Timeline property editor clean up

### PR DESCRIPTION
**Pull request by lauriLukkarinen**
Resolves #326

### CODE REVIEW

File: src/components/content-editor/dynamic-resource-editor.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      break;
+      case "key":
+        draft.key = value as string;
+      break;
+      default:
+      break;
+    }
+  });
+
+  console.log(updatedParams);
```
Comment (jarinykanen) [inline src/components/content-editor/dynamic-resource-editor.tsx:112]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```suggestion
```

Comment (jarinykanen) [inline src/components/content-editor/dynamic-resource-editor.tsx:112]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```suggestion
```

File: src/components/content-editor/resource-editor.tsx [hunk 2]
```diff
@@ -51,162 +47,208 @@ class ResourceEditor extends React.Component<Props, State> {
...
    */
-  private renderModeButton = () => {
-    const { resource } = this.props;
+  private renderModeSelect = () => {
+    const { classes, resource } = this.props;
+
+    const selectOptions = [
+      PageResourceMode.Static,
+      PageResourceMode.Dynamic,
+      PageResourceMode.Scripted
```
Comment (jarinykanen) [inline src/components/content-editor/resource-editor.tsx:68]: Move to content-editor/constants.ts

File: src/components/screens/content-editor-screen.tsx [hunk 3]
```diff
@@ -622,29 +669,31 @@ class ContentEditorScreen extends React.Component<Props, State> {
...
-    });
+      return filtered;
+      // return (
+      //   <ResourceEditor
+      //     key={ foundResource.id }
+      //     resource={ foundResource }
+      //     resourceIndex={ resourceIndex }
+      //     onUpdate={ this.onUpdateResource }
+      //   />
+      // );
```
Comment (jarinykanen) [inline src/components/screens/content-editor-screen.tsx:693]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

**Review by jarinykanen — CHANGES_REQUESTED** [review]
Check comments

File: src/components/screens/content-editor-screen.tsx (hunk 3 above)
Comment (lauriLukkarinen) [inline src/components/screens/content-editor-screen.tsx:693]:
```suggestion
```

**Pull request merged**